### DISCUSSION TITLE
No last mission

### DISCUSSION THREAD

**Discussion by Hakanan28** [IBAX-io/go-ibax#1717] [Bug Submission]
Hello. Have a nice day. Not accepting the last 5000 points retweet post. Even though I retweet it, it is not accepted. May I know your reason?

**Comment by alisabanoglu** [comment]
bir saattir deniyorum ama bende sürekli hata alıyorum....

**Reply by Hakanan28** [reply]
Aynen. Düzelecek dendi ama ses seda yok

**Reply by ahmetnur99** [reply]
İnşallah süre bitmeden hallolur

**Reply by Serhat67** [reply]
neden  bu sorun biran önce düzeltilmiyo  süre azaldı

**Reply by Hakanan28** [reply]
Katılıyorum. Bir arkadaş almış ödülü sabah erkenden. Şimdi yoğunluk deniliyor. Sistemin düzenlenmesi şart. Büyük bir proje olması için. Ki projesi sağlam olduğuna inanıyorum. Ama sitede düzeltmeler olması şart
